lighthouse-viewer/app/styles/viewer.css

```diff
@@ -124,3 +124,7 @@
 .lh-tools__dropdown .lh-tools--gist {
   display: block !important;
 }
+
+.lh-loading {
```

Comment:
move this up to the `.viewer-placeholder-inner` styles.

`.viewer-placeholder-inner.loading` probably?